frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1001,6 +1045,31 @@ private void prepareLicenseInfo(RenderRequest request, RenderResponse response)
         }
     }
 
+    private Set<String> fetchReleaseRelationships(List<ProjectLink> mappedProjectLinks) {
+        return mappedProjectLinks.stream().map(ProjectLink::getLinkedReleases).flatMap(List::stream)
+                .map(ReleaseLink::getReleaseRelationship).map(ThriftEnumUtils::enumToString)
+                .collect(Collectors.toSet());
+    }
+
+    private Set<String> fetchIcludedReleaseRelationships(String projectId) {
+        Set<String> excludedRelationships = Sets.newHashSet();
+        try {
+            final AttachmentService.Iface attachmentClient = thriftClients.makeAttachmentClient();
+            List<AttachmentUsage> usages = attachmentClient.getUsedAttachments(Source.projectId(projectId), null);
```

Comment:
Instead of fetching all the AttachmentUsages , please try to fetch only LicenseInfo  related AttachmentUsage .

Reply:
Done